app/src/main/java/org/mozilla/fenix/ext/TabCollection.kt

```diff
@@ -22,3 +23,17 @@ fun TabCollection.getIconColor(context: Context): Int {
     iconColors.recycle()
     return color
 }
+
+/**
+ * Returns the new default name recommendation for a collection
+ *
+ * Algorithm: Go through all collections, make a list of their names and keep only the default ones.
+ * Then get the numbers from all these default names, compute the maximum number and add one.
+ */
+fun List<TabCollection>.getDefaultCollectionNumber(): Int {
```

Comment:
nit: is this necessary? I'm not sure what UX asked for here, but it feels pretty reasonable to just name the collection based on the total list of collections: e.g. I have 7 collections, I make a new one, and it suggests "Collection 8"

Reply:
oh we already had this in another file. Okie 🤷‍♂️